da/avail/avail.go

```diff
@@ -155,11 +155,12 @@ func (c *DataAvailabilityLayerClient) GetClientType() da.Client {
 }
 
 // RetrieveBatch retrieves batch from DataAvailabilityLayerClient instance.
-func (c *DataAvailabilityLayerClient) RetrieveBatches(dataLayerHeight uint64) da.ResultRetrieveBatch {
-	blockHash, err := c.client.GetBlockHash(dataLayerHeight)
+func (c *DataAvailabilityLayerClient) RetrieveBatches(daMetaData *da.DASubmitMetaData) da.ResultRetrieveBatch {
```

Comment:
something off with the naming, we pass `DASubmitMetaData` to *Retrieve* function

Reply:
I named DASubmitMetadata, the DA metadata result of the submission, and used for retrieving the data.
I named DACheckMetaData, the DA metadata resulting of the availability checks including inclusion proofs for disputes. 
I can change the naming if you think is not clear enough